### ISSUE TITLE
CSS Improvement Suggestion for Compact Mode

### ISSUE THREAD

**Issue by jstueckrath** [gatanaso/multiselect-combo-box#62]
I noticed that the compact mode layout is not consistent with the standard layout and would therefore suggest the following CSS changes (which I also used in my project):

```CSS
[part="compact-mode-label"] {
	margin: var(--lumo-space-xs);
	color: var(--lumo-body-text-color);
	font-weight: 500;
	padding: 0 calc(0.375em + var(--lumo-border-radius)/4 - 1px);
}
```

The color and font-weight above are the same as the ones used in non-compact mode. The margin will prevent the combo box from being larger in compact mode compared to non-compact mode (as it currently is, see below). The padding makes no difference in my example, but I took it from the Textfield definition and think it is more flexible.

See the changes below compared to other fields (layouted by Chromium). Note that I am still using the Vaadin standard layouts.

Before:
![Layout Compare Before](https://user-images.githubusercontent.com/38353563/77298024-76d46700-6cea-11ea-937e-4a0d132fa73e.PNG)

After:
![Layout Compare After](https://user-images.githubusercontent.com/38353563/77298034-7b008480-6cea-11ea-977f-8eddcbf5705c.PNG)

**Comment by gatanaso** [comment]
Hi @jstueckrath 

Thanks for the great suggestion, especially for the before/after which really nicely shows the improvement in the component. This is a great idea and definitely something that could be introduced. One thing to consider would be the material theme, though I assume the same approach could be applied there.

**Comment by jstueckrath** [comment]
You're welcome, I am only using Lumo and did not test it with Material, but you are probably right that it can be improved similarly.

**Comment by gatanaso** [comment]
Hi @jstueckrath,

I am working on improving the CSS based on your suggestions here, but I cannot reproduce your Before example. Would it be possible that you share a small reproducible project where I can see the odd height in action? 

Thanks,
Goran

**Comment by jstueckrath** [comment]
Sorry, but I did not have time to produce a small running example. Somehow Vaadin did not load all necessary libraries properly. I will have to take a look at that next week.

P.S.: I confirmed that the project I am working on still has the problem. It uses Vaadin 14.1.17 and MultiselectComboBox 2.3.1. Maybe the layout changed in newer versions.

**Comment by jstueckrath** [comment]
Hello @gatanaso 
Sorry, but I did not manage to create a minimal project for this issue. Vaadin starts, but displays the following Javascript problems:
```
(TypeError) : Cannot read property 'initLazy' of undefined
(TypeError) : Cannot read property 'updateSize' of undefined
(TypeError) : Cannot read property 'set' of undefined
(TypeError) : Cannot read property 'confirm' of undefined
```
I tried version 2.3.1 and 2.5.0 of MultiselectComboBox, both with the same result. I do not understand where the problem is. The package-lock.json in the project root and the package.json in the target folder both show "multiselect-combo-box" as a Javascript dependency, but it seem that it is not properly loaded. Have you encountered this problem before?

**Comment by gatanaso** [comment]
Hi @jstueckrath. No I have not encountered this problem before. Which version of Vaadin are you using?

**Comment by jstueckrath** [comment]
Hi @gatanaso, for that attempt I used Vaadin 14.1.17, but I will also try it with a newer version. Strangely I have a project with exactly that combination of versions which runs just fine... It must be some kind of Problem with the project setup. I will notify you once I fixed the issue.

**Comment by jstueckrath** [comment]
Hi @gatanaso, I finally found time and worked around the type errors I encountered earlier. Here is an example project for the before/after examples I sent in the beginning:

[multiselect-combo-box-issue62.zip](https://github.com/gatanaso/multiselect-combo-box/files/5028884/multiselect-combo-box-issue62.zip)

If you reactivate the `@CssImport` in the `MainView` class, you will see the "after" effect (required another run of `mvn clean package` for me). The example uses Vaadin 14.2.3 and MultiselectComboBox 2.5.0.
